spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "posts/show", type: :view do
+  before(:each) do
+    @post = assign(:post, Post.create!(
+      title: "Title",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstArgumentIndentation: Indent the first argument one step more than Post.create!(.